customChatWidget/src/containers/ChatWidget/index.js

```diff
@@ -178,15 +171,45 @@ const ChatWidget = ({
             },
         }
         log('Params to initiate chat connection: ', params);
-
         try {
             window.connect.ChatInterface.initiateChat(params, successHandler, failureHandler);
+            setChatInitialized(true);
         }
         catch (e) {
             error('window.connect.ChatInterface.initiateChat');
             error(e);
+            setChatInitialized(false);
+        }
+    }
+
+    // eslint-disable-next-line
+    useEffect(() => {
```

Comment:
[nit] Do we need the log ?

Reply:
Since this is a ui-example, these logs are used for debugging purposes.  Could be useful for builders to see the `useEffect` log printed to indicate when this code is triggered.  Also this PR doesn't introduce this log, this was  existing from the old `useEffect` block that was here.